pmm/pmm2-release.groovy

```diff
@@ -188,7 +330,6 @@ pipeline {
                         DOCKER_MID="\$TOP_VER.\$MID_VER"
                         sg docker -c "
                             set -ex
-                            aws ecr-public get-login-password --region us-east-1 | docker login -u AWS --password-stdin public.ecr.aws/e7j3v3n0
```

Comment:
how you are going to push image to repo without docker login?

Reply:
We push the image to the Docker hub because of it's public image. We just don't need AWS registry here.

Reply:
yes but you need to do docker login to dockerhub. otherwise anyone in the world would be able to push any image into any repo.

Reply:
We have it slightly above:
https://github.com/Percona-Lab/jenkins-pipelines/blob/86331e575c9ad5632d20f91fa62f162b8327f2d7/pmm/pmm2-release.groovy#L311